src/util/reactTestingLibraryExtended.js

```diff
@@ -13,8 +13,10 @@ const AllTheProviders = ({ children }) => {
 const customRender = (ui, options) =>
   render(ui, { wrapper: AllTheProviders, ...options })
 
+// CAUSING PROBLEMS with babel + standard + eslint tool chain
```

Comment:
TBD on this one, but this branch does not yet rely on this file.

Reply:
dont we use this in @thomasgwatson  code?

Reply:
Yep, and once I merge it up hopefully it'll just work. There is a test in here that relies on the remainder of his changes to this file that I need for it to run. It'll be all resolved in the merge-up I'll do tomorrow now that it's in dev, and fingers crossed that this issue resolves itself. If not I'll have to figure go to poke at it more.